Gemfile

```diff
@@ -22,6 +22,7 @@ group :development, :test do
   gem 'omniauth-github',        git: 'https://github.com/intridea/omniauth-github'
   gem 'omniauth-google-oauth2', git: 'https://github.com/zquestz/omniauth-google-oauth2'
   gem 'rack-cors', require: 'rack/cors'
+  gem 'sqlite3', '~> 1.3.6'
```

Comment:
why adding sqlite?

Reply:
had trouble running specs on my dev machine, but can be removed of course if you don't see it fit.

```suggestion
```

Reply:
if you can, try to remove it from here and other places, if you have issues that can happen to other developers, we can keep it.

Reply:
I removed it now, tests work fine without it now, too. I think it is because the sqlite version is specified in other gemfiles now. In them it is needed though, since otherwise it will install sqlite >= 1.4 which breaks with current rails versions.